src/bot.js

```diff
@@ -22,6 +22,10 @@ let initializeBot = () => {
 		logger.info(user.username + ' - (' + user.id + ')');
 	});
 
+	//var currentInhouse;
+	var serverInhouses = {};
```

Comment:
Prefer `let` over `var` unless you need hoisting. 

https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var#targetText=The%20main%20difference%20is%20the,outside%20the%20loop%20for%20example.&targetText=This%20is%20unlike%20the%20var,function%20regardless%20of%20block%20scope.